app/forms/taxon_form.rb

```diff
@@ -1,5 +1,5 @@
 class TaxonForm
-  attr_accessor :title, :parent, :content_id, :base_path
+  attr_accessor :title, :taxon_parents, :content_id, :base_path
```

Comment:
Minor thing, but the naming feels a little redundant here. This may as well be `parents`. If this is about naming confusion with the existing `parent` field, perhaps `parent_taxons` would be better still. This way you're explicitly stating that the parent items are taxons themselves.